pydm/tests/widgets/test_colormaps.py

```diff
@@ -20,16 +20,6 @@ def test_construct():
     Expecations:
     The default values are assigned to the attributes correctly.
     """
-    pydm_colormap = PyDMColorMap()
```

Comment:
That was just claiming coverage for 1061-1067... another not much useful test since all values are class variables we don't need to instantiate it and we are accessing them on the lines below.

Reply:
Ohh ok, thank you